website/templates/profile/addons.mako

```diff
@@ -1,8 +1,14 @@
 <%inherit file="base.mako"/>
-<%def name="title()">Configure Add-ons</%def>
+<%def name="title()">Configure External Accounts </%def>
 <%def name="content()">
+<style>
+.addon-icon {
+    width: 20px;
+}
+</style>
+
 <% from website import settings %>
-<h2 class="page-header">Configure Add-ons</h2>
+<h2 class="page-header">Configure External Accounts</h2>
```

Comment:
I think I mentioned this in a previous comment, but I think Configure External Accounts could be changed to "Configure Add-on Accounts" or "Configre Add-on External Accounts" for clarity